installimg/8.2.updates/etc/systemd/system/sshd.service.d/installer.conf

```diff
@@ -0,0 +1,5 @@
+[Unit]
```

Comment:
My choice to leave it here was because it is literally zero-cost to get the file as part of the file copying, whereas going to the SRPM requires extra work that will eventually be reverted, for very little benefit.